manifests/nodejs.yml

```
@@ -1251,7 +1251,7 @@ tests/:
       Test_Config_TraceEnabled: *ref_4_3_0
       Test_Config_TraceLogDirectory: missing_feature
       Test_Config_UnifiedServiceTagging: *ref_5_25_0
-      Test_Stable_Config_Default: *ref_5_27_0
+      Test_Stable_Config_Default: irrelevant (temporarily ignored)
```

Wdyt about having this on a future release instead? Note that we'd need to include this on the top of this yml as well

```suggestion
      Test_Stable_Config_Default: *ref_5_62_0
```